builtin/logical/ssh/backend_test.go

```diff
@@ -105,6 +105,13 @@ SeKWrUkryx46LVf6NMhkyYmRqCEjBwfOozzezi5WbiJy6nn54GQt
 `
 )
 
+type CertValidityValidationMethod int
+
+const (
+	ValidateTTL            CertValidityValidationMethod = 0
+	ValidateAfterAndBefore CertValidityValidationMethod = 1
```

Comment:
This `ValidateAfterAndBefore` enum is used to apply new logic for validation of the `ValidAfter` and `ValidBefore` values when they are specified by the new `valid_after` and `valid_before` parameters